### PR TITLE
Add some small point cloud files for unit tests

### PR DESCRIPTION
**Pull request by nyalldawson**
Adds some small (~200 points) point cloud files in entwine and las format for use in future unit tests.

@wonder-sk @PeterPetrik not sure if you had alternatives planned -- these ones are basic, but should do the job for generic tests

### CODE REVIEW

File: scripts/spell_check/check_spelling.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@
 EXCLUDE_SCRIPT_LIST='(\.(xml|svg|sip|pl|sh|qgs|badquote|cmake(\.in)?)|^(debian/copyright|cmake_templates/.*|tests/testdata/labeling/README.rst|tests/testdata/font/QGIS-Vera/COPYRIGHT.TXT|doc/NEWS\.html|debian/build/))$'
 
 # always exclude these external files
-EXCLUDE_EXTERNAL_LIST='(resources/server/api/ogc/static/landingpage/js/.*)$'
+EXCLUDE_EXTERNAL_LIST='(resources/server/api/ogc/static/landingpage/js/.*|/home/nyall/dev/QGIS/tests/testdata)$'
```
Comment (PeterPetrik) [inline scripts/spell_check/check_spelling.sh:29]: this doesn't looks correct :)

**Review by PeterPetrik — APPROVED** [review]
Looks good. 

also I like (for ept)
https://github.com/PDAL/PDAL/tree/master/test/data/ept/ellipsoid-zstandard

and for laz/las there are bunch of nice files here too
https://github.com/PDAL/PDAL/tree/master/test/data

**Comment by PeterPetrik** [comment]
@nyalldawson can you add screenshot of the file so we see what it is?

File: scripts/spell_check/check_spelling.sh (hunk 1 above)
Comment (nyalldawson) [reply scripts/spell_check/check_spelling.sh:29]: :rofl:

**Review by PeterPetrik — APPROVED** [review]

**Comment by nyalldawson** [comment]
@PeterPetrik 

Here's what it looks like:

image

**Pull request merged**